helix-core/src/test/java/org/apache/helix/integration/TestZkConnectionLost.java

```diff
@@ -66,6 +66,8 @@
   private ClusterSetup _setupTool;
   private HelixZkClient _zkClient;
 
+  private static final int RESTART_CNT = 2;
```

Comment:
It was 4.

Reply:
This some times has timeout even with enlarging the default timeout of unit test. Zk operation is very slow in Github.

Reply:
Let me change back to 4 to see how it goes.